cmd/antrea-agent/options.go

```diff
@@ -453,10 +453,11 @@ func (o *Options) validateK8sNodeOptions() error {
 			}
 		}
 	}
-	if (features.DefaultFeatureGate.Enabled(features.Multicluster) || o.config.Multicluster.Enable) &&
-		encapMode != config.TrafficEncapModeEncap {
-		// Only Encap mode is supported for Multi-cluster feature.
-		return fmt.Errorf("Multicluster is only applicable to the %s mode", config.TrafficEncapModeEncap)
+	if features.DefaultFeatureGate.Enabled(features.Multicluster) && !o.config.Multicluster.Enable {
+		return fmt.Errorf("The multicluster.enable config option must be true to enable Antrea Multi-cluster feature")
+	}
+	if !features.DefaultFeatureGate.Enabled(features.Multicluster) && o.config.Multicluster.Enable {
```

Comment:
No, I do not mean WG for Multi-cluster traffic. I meant how if WG is enabled for the cluster - it can be viewed as a new mode - not encap, noEncap.

Reply:
I will double check this with WireGuard encryption.

Reply:
Hi @jianjuns I checked the MC with WireGuard, it will not work since the traffic won't go through `antrea-tun0` anymore. I am planing to add an option validation step to fail the agent if it's `encap` with `WireGuard` encryption.
what's your thought?

Reply:
That sounds good to me - fail validation if WireGuard is configured.